### PR TITLE
Obada - I put my name and put a picture of the layout design

### PR DESCRIPTION
**Pull request by ObadaMoustafa**
**Obada** - I put my name and **put a picture of the layout design** it's not professional but we can start using it just to work around

### CODE REVIEW

**Comment by claudiacaus** [comment]
@ObadaElSharbatly I think it's better to add the full name.

File: README.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ index.html
 - `views` this contains code to define what the DOM will look like. They will create the DOM element and give it back. They should never read from/write to the dom, that is what the handlers do.
 - `data.js` this is our data model. Anything we need to store in the browser we place inside the data file
 
+# Project layout design:
+it's not profissional :D but just can use it to start
+![layout design image](tenQProject.jpg)
```
Comment (claudiacaus) [inline README.md:44]: There are misspelling errors in the layout, "Ninja", but well done!

File: README.md [hunk 2]
```diff
@@ -67,5 +71,7 @@ So what should be built? Below is a collection of user stories you can choose fr
 
 # Ninja's Team
 The Starter Project is investigated by a bunch of Ninja's to implement some base functionality, some extended nice to have functionality and to make it look nice. 
-This group of Ninja's consists of: 
+This group of Ninja's consists of:
+
+Obada ELSharbatly
```
Comment (claudiacaus) [inline README.md]: Well done!

**Review by claudiacaus — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ObadaMoustafa) [reply README.md:44]: @claudiadewindt 
I think it gonna work after merging because it works on my branch

**Review by claudiacaus — APPROVED** [review]

**Comment by Amar-Mahdy** [comment]
well done obada

**Pull request merged**